react/src/components/PayButton/PayButton.stories.tsx

```diff
@@ -46,14 +51,14 @@ UsdAmount.args = {
   amount: 5,
   currency: 'USD',
   text: '$5',
-  hoverText: 'Pay with BCH',
+  hoverText: 'Pay with '.concat(defaultCurrency),
```

Comment:
Any reason you decided to go with `.concat` instead of template strings e.g. `` `Pay with ${defaultCurrency}` ``? I'm fine with the way it is, just out of curiosity